File: LP/C/odds.c
#include "stdio.h"

void writeOdds(int number);

int main(void)
{
    int num;

    printf("Digite o número de ímpares: ");
    scanf("%d", &num);

    writeOdds(num);

    return 0;
}

void writeOdds(int number)
{
    for (int i = 1; i <= number; i++)
    {
        printf("%d ", 2 * i + 1);
    }

    printf("\n");
}

File: LP/GO/odds.go
package main

import (
	"fmt"
)

func writeOdds(n int) {
	for i := 1; i <= n; i++ {
		fmt.Printf("%d ", 2*i+1)
	}
	fmt.Println()
}

func main() {
	var num int
	fmt.Print("Digite o número de ímpares: ")
	fmt.Scan(&num)

	writeOdds(num)
}
